apps/api/src/controllers/auth.ts

```
@@ -111,15 +196,22 @@ export async function supaAuthenticateUser(
-    teamId = "preview";
+    throw new Error("Unauthenticated Playground calls are temporarily disabled due to abuse. Please sign up.");
+
+    if (mode == RateLimiterMode.CrawlStatus) {
+      rateLimiter = getRateLimiter(RateLimiterMode.CrawlStatus, token);
+    } else if (mode == RateLimiterMode.ExtractStatus) {
+      rateLimiter = getRateLimiter(RateLimiterMode.ExtractStatus, token);
+    } else {
+      rateLimiter = getRateLimiter(RateLimiterMode.Preview, token);
+    }
+    teamId = `preview_${iptoken}`;
+    plan = "free";
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)